curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646f516dbfc1342495515625.md

```diff
@@ -0,0 +1,572 @@
+---
+id: 646f516dbfc1342495515625
+title: Step 87
+challengeType: 0
+dashedName: step-87
+---
+
+# --description--
+
+Rotate the bottom-right whisker at 3 degrees.
```

Comment:
Since this is the last step, we should add a short message at the end acknowledging that. Just like the other projects do.

```suggestion
Rotate the bottom-right whisker at 3 degrees.

With this final step, your cat painting is now complete.
```